### PR TITLE
Add nf_conntrack counter support

### PR DESCRIPTION
**Pull request by jimmystewpot**
This PR adds support to read in the current count and the max count settings from the netfilter files in /proc. This is useful for people who need to query statistics iptables based NAT boxes.

### CODE REVIEW

File: net/net_linux.go [hunk 1]
```diff
@@ -160,3 +160,31 @@ func NetProtoCounters(protocols []string) ([]NetProtoCountersStat, error) {
 	}
 	return stats, nil
 }
+
+// NetFilterCounters returns iptables conntrack statistics
+// the currently in use conntrack count and the max.
+// If the file does not exist or is invalid it will return nil.
+func NetFilterCounters() ([]NetFilterStat, error) {
+	countfile := "/proc/sys/net/netfilter/nf_conntrack_count"
```
Comment (shirou) [inline net/net_linux.go]: Could you change to use `common.HostProc`?

File: net/net.go [hunk 2]
```diff
@@ -64,6 +64,11 @@ type NetInterfaceStat struct {
 	Addrs        []NetInterfaceAddr `json:"addrs"`
 }
 
+type NetFilterStat struct {
+	ConnTrackCount	int64	`json:"conntrackcount"`
+	ConnTrackMax	int64	`json:"conntrackmax"`
```
Comment (shirou) [inline net/net.go]: Could you change these json name to snake_case, such as `conn_track_max` ?
(I know JSON use camelCase on their name, but I had not know that at first, and other APIs already have snake_case.)

**Comment by shirou** [comment]
Thank you for the awesome PR! I have made some comments. And could you add this cool feature to the README.rst? 
After that, I will merge soon. Thank you!

**Comment by jimmystewpot** [comment]
Thanks for the feedback.
I have made the changes that you requested and updated the documentation. If this is successful I will have more additions coming over the next few weeks.

**Comment by jimmystewpot** [comment]
@shirou is there anything missing from this PR now?

**Comment by shirou** [comment]
Sorry, I have not noticed your update. It looks great. Thank you very much!

**Pull request merged**